agr_indexer/src/main/java/org/alliancegenome/indexer/indexers/curation/interfaces/DiseaseDocumentInterface.java

```diff
@@ -0,0 +1,30 @@
+package org.alliancegenome.indexer.indexers.curation.interfaces;
+
+import java.util.HashMap;
+
+import org.alliancegenome.curation_api.model.document.es.DiseaseSummaryDocument;
+import org.alliancegenome.curation_api.response.SearchResponse;
+import org.alliancegenome.curation_api.view.View;
+
+import com.fasterxml.jackson.annotation.JsonView;
+
+import jakarta.ws.rs.Consumes;
+import jakarta.ws.rs.DefaultValue;
+import jakarta.ws.rs.POST;
+import jakarta.ws.rs.Path;
+import jakarta.ws.rs.Produces;
+import jakarta.ws.rs.QueryParam;
+import jakarta.ws.rs.core.MediaType;
+
+@Path("/disease/document")
+@Produces(MediaType.APPLICATION_JSON)
+@Consumes(MediaType.APPLICATION_JSON)
+public interface DiseaseDocumentInterface {
+	@POST
+	@Path("/summary")
+	@JsonView({ View.DiseaseSummaryDocument.class })
+	SearchResponse<DiseaseSummaryDocument> findSummary(
+		@DefaultValue("0") @QueryParam("page") Integer page,
+		@DefaultValue("10") @QueryParam("limit") Integer limit,
```

Comment:
@VarunReddy1111 you should be able to use the interface in the curation system vs creating a new one here.